prow/flagutil/config/config.go

```diff
@@ -59,6 +61,7 @@ func (o *ConfigOptions) AddFlags(fs *flag.FlagSet) {
 	fs.StringVar(&o.SupplementalProwConfigsFileNameSuffix, "supplemental-prow-configs-filename-suffix", "_prowconfig.yaml", "Suffix for additional prow configs. Only files with this name will be considered")
 	fs.IntVar(&o.InRepoConfigCacheSize, "in-repo-config-cache-size", 1000, "Cache size for ProwYAMLs read from in-repo configs. Each host receives its own cache.")
 	fs.StringVar(&o.InRepoConfigCacheDirBase, "cache-dir-base", "", "Directory where the repo cache should be mounted.")
+	fs.IntVar(&o.TimeoutListingProwJobs, "timeout-listing-prowjobs", 60, "Timeout for listing prowjobs in seconds.")
```

Comment:
I don't think we should be changing the default value to 60. 
- I don't think it will break anyone, but it is possible that it could affect deployment strategy configuration. 
- More importantly, 30 seconds is a more reasonable default. We're only raising this to 60 seconds in our use case as a temporary performance mitigation.

Reply:
It looks like we actually had a different default for horologium. If we want to keep this we could add a default flag value field to the new flag bundle (mentioned in my other comment) like we do here with [`NOInClusterConfigDefault`](https://github.com/kubernetes/test-infra/blob/1dcc1a65ba4b4f47b19fcca8121f38256164bbe5/prow/flagutil/kubernetes_cluster_clients.go#L65).

Reply:
The example you linked has 2 fields:
```
	noInClusterConfig        bool
	NOInClusterConfigDefault bool
```

so are you suggesting that we can have the same thing here for TimeoutListingProwJobs? (I.e., add a second TimeoutListingProwJobsDefault field?)

Reply:
Yes, the first is the actual flag variable and the second (with the "Default" suffix) is the default value to use for the flag. We can then override the default flag value of 30 in the main function of horologium by setting `.TimeoutListingProwJobsDefault = 60` before we add the flags.